source/playbooks/common/parse_input.py

```diff
@@ -81,11 +81,17 @@ def _get_resource_id(self, parse_id_pattern, resource_index):
                 self.resource_id = identifier_match.group(resource_index)
             else:
                 exit(f'ERROR: Invalid resource Id {identifier_raw}')
-            return
 
     def _get_standard_info(self):
+        standard_arn_pattern_tuple = (r'^arn',
+            r'(?:aws|aws-cn|aws-us-gov)',
+            r'securityhub',
+            r'(?:[a-z]{2}(?:-gov)?-[a-z]+-\d)',
+            r'\d{12}',
+            r'subscription/(.*?)/v/(\d+\.\d+\.\d+)/(.*)/finding/(?i:[0-9a-f]{8}-(?:[0-9a-f]{4}-){3}[0-9a-f]{12})$',)
+        standard_arn_pattern = ":".join(standard_arn_pattern_tuple)
```

Comment:
nit: this could just be
```
standard_arn_pattern = ":".join(r'^arn',
    r'(?:aws|aws-cn|aws-us-gov)',
    r'securityhub',
    r'(?:[a-z]{2}(?:-gov)?-[a-z]+-\\d)',
    r'\\d{12}',
    r'subscription/(.*?)/v/(\\d+\\.\\d+\\.\\d+)/(.*)/finding/(?i:[0-9a-f]{8}-(?:[0-9a-f]{4}-){3}[0-9a-f]{12})$',)
```

which may or may not make it clearer that each row is joined by a semicolon.

is such a minor difference though